src/ui/components/TransferGrain.js

```diff
@@ -18,90 +17,151 @@ export type Props = {|
   +setLedger: (Ledger) => void,
 |};
 
+const useStyles = makeStyles({
+  root: {
+    width: "1100px",
+    margin: "0 auto",
+    background: "white",
+    padding: "0 5em 5em",
+  },
+  arrowBody: {
+    width: "300px",
+    marginLeft: "20px",
+    background: "#C9CED2",
+    padding: "5px 20px",
+  },
+  triangle: {
+    width: 0,
+    height: 0,
+    background: "white",
+    borderTop: "30px solid transparent",
+    borderBottom: "30px solid transparent",
+    borderLeft: "30px solid #C9CED2",
+  },
+  dropdown: {
+    width: "300px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    borderRadius: "500px",
+    background: "#F2F5F7",
+  },
+  centerRow: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  third: {
+    width: "300px",
+    height: "50px",
+    margin: "30px 15px",
+  },
+  half: {
+    width: "450px",
+    height: "50px",
+    margin: "30px",
+  },
+  memo: {
+    width: "960px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    paddingLeft: "20px",
+    color: "black",
```

Comment:
Instead of setting the border like this, just use the `variant="outlined"` prop of the [`TextField`](https://material-ui.com/components/text-fields/#textfield) component. Should probably not use the Input component directly from material-ui